### PR TITLE
SBNotification Parsing Fix

### PR DESCRIPTION
**Pull request by drw86312**

### CODE REVIEW

File: Projects/Library/CocoaBloc/API/SBClient+Notification.m [hunk 1]
```diff
@@ -21,9 +21,9 @@ - (RACSignal *)getNotificationsForAccount:(SBAccount *)accountOrNil parameters:(
         p[@"account_id"] = accountOrNil.identifier;
     }
     
-    return [[[self rac_GET:@"users/me/notifications" parameters:[self requestParametersWithParameters:p]]
-                cb_deserializeArrayWithClient:self modelClass:[SBNotification class] keyPath:@"data"]
-                setNameWithFormat:@"Get notifications (account: %@)", accountOrNil];
+    return [[[[self rac_GET:@"users/me/notifications" parameters:[self requestParametersWithParameters:p]] map:^id(NSDictionary *response) {
+        return @{@"data" : response[@"data"][@"notifications"]};
+    }] cb_deserializeArrayWithClient:self modelClass:[SBNotification class] keyPath:@"data"] setNameWithFormat:@"Get notifications (account: %@)", accountOrNil];
```
Comment (joshholat) [inline Projects/Library/CocoaBloc/API/SBClient+Notification.m]: Man, iOS indentation makes this shit so hard to decipher what's going on ha

**Comment by joshholat** [comment]
Yo @drw86312 can you provide a description in the PR of what this fixes? I'm assuming it's just properly getting the notifications from the response since they are nested within the `data` key.

**Comment by drw86312** [comment]
PR retrieves notifications array nested in the 'data' key.

**Comment by JRHeaton** [comment]
Looks good. Glad we wound up with a solution without polluting CB with a super-specific signal operator 👍

**Pull request merged**